attachment_synchronize/models/task.py

```diff
@@ -126,6 +129,8 @@ def run_import(self):
         backend = self.backend_id
         filenames = backend._list(
             relative_path=self.filepath, pattern=self.pattern)
+        if self.check_duplicated_files:
+            filenames = self._file_to_import(filenames)
```

Comment:
@florian-dacosta isn't filenames a list of names? am I right?

Reply:
AFAIK it is, yes